tests/foreman/maintain/test_advanced.py

```diff
@@ -70,19 +73,27 @@
     'satellite-capsule-6.14-for-rhel-8-x86_64-rpms',
     'satellite-maintenance-6.14-for-rhel-8-x86_64-rpms',
 ] + common_repos
+cap_615_repos = [
+    'satellite-capsule-6.15-for-rhel-8-x86_64-rpms',
+    'satellite-maintenance-6.15-for-rhel-8-x86_64-rpms',
+] + common_repos
 
 sat_repos = {
```

Comment:
Can we flatten this dict into array as we test just single satellite version per each robottelo branch

Reply:
yep, makes sense.

Reply:
done